indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -577,30 +577,60 @@ th.i-table {
     }
 }
 
-.recipients {
-    @extend .flexrow;
-    justify-content: space-between;
+#form-group-recipients {
+    position: relative;
 
-    .static-text-widget {
-        width: inherit;
+    .recipients-box {
+        @include border-all($pastel-gray);
+
+        display: inline-block;
+        padding-left: 5px;
+        border-radius: 2px;
+        margin: 0 1em 0.7em 1em;
+        width: 75%;
+
+        button {
+            position: absolute;
+            right: 30px;
+            top: 2px;
+            background: none;
+            border: 0;
+            padding: 5px;
+        }
     }
 
-    .static-text-widget p {
+    .recipients {
+        @extend .flexrow;
+        justify-content: space-between;
+        width: 68%;
```

Comment:
These values are kinda ugly... There might be a way for flexbox to handle these...